packages/canon/src/components/Heading/Heading.tsx

```diff
@@ -14,46 +14,41 @@
  * limitations under the License.
  */
 
-import { forwardRef } from 'react';
+import { forwardRef, useRef } from 'react';
 import clsx from 'clsx';
 import { useResponsiveValue } from '../../hooks/useResponsiveValue';
-
+import { useRender } from '@base-ui-components/react/use-render';
 import type { HeadingProps } from './types';
 
 /** @public */
 export const Heading = forwardRef<HTMLHeadingElement, HeadingProps>(
   (props, ref) => {
     const {
-      children,
       variant = 'title1',
-      as = 'h1',
+      color = 'primary',
       truncate,
       className,
+      render = <h1 />,
```

Comment:
Element version had slightly worse perf right? Thinking that for a default like this it's best to use the format with the least overhead. No biggie though